docs/changelog/99029.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 99029
```

Comment:
I don't think this change needs a changelog entry. You can remove this by updating the tag of this PR to be `>non-issue` rather than `>enhancement`.

Reply:
We were providing indices before in the response, but removed them when we stopped storing them. This PR brings them back, so it's not really an enhancement but keeping the status quo.